packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -782,4 +784,55 @@ export class PhoenixObjects {
 
     return object;
   }
+
+  /**
+   * Type for drawing irregular calorimeter cell comprising 8 xyz vertices in arbitrary geometry.
+   * @param irrCells Properties of irregular calo cell.
+   * @param irrCells.layer Calorimeter layer
+   * @param irrCells.vtx Flattened list of 8 vertex coordinates (24 floats)
+   * @param irrCells.color [R,G,B] integer list
+   * @param irrCells.opacity value from 0 to 1
+   * @returns the cell
+   */
+  public static getIrregularCaloCell(irrCells: {
+    type: any;
+    layer: number;
+    vtx: any;
+    color: string;
+    opacity: any;
+  }): Object3D {
+    const verticesOfCube = [];
+    for (let i = 0; i < 24; i += 3) {
+      verticesOfCube.push(
+        new Vector3(irrCells.vtx[i], irrCells.vtx[i + 1], irrCells.vtx[i + 2])
+      );
+    }
+    const geometry = new ConvexGeometry(verticesOfCube);
+    const cell_color = new Color(
+      'rgb(' +
+        irrCells.color[0].toString() +
+        ',' +
+        irrCells.color[1].toString() +
+        ',' +
+        irrCells.color[2].toString() +
+        ')'
+    );
+
+    // material
+    const material = new MeshPhongMaterial({
+      color: cell_color,
+      transparent: true,
+      opacity: irrCells.opacity,
+    });
+
+    // object
+    const cell = new Mesh(geometry, material);
+    cell.userData = Object.assign({}, irrCells);
+    cell.name = 'IrregularCaloCell';
+
+    // Setting uuid for selection from collections info
+    irrCells.uuid = cell.uuid;
```

Comment:
There is no uuid in irrCells ! So my ci (LHCb's deployment one) gives the following error : 
```
src/loaders/objects/phoenix-objects.ts(834,14): error TS2339: Property 'uuid' does not exist on type '{ type: any; layer: number; vtx: any; color: string; opacity: any; }'.
```
I do not get why we did not see it in the official repo ! Can you fix it ?